src/Compilers/Core/Portable/CommandLine/CommandLineParser.cs

```diff
@@ -1064,11 +1071,10 @@ internal static void ParseSeparatedStrings(ReadOnlyMemory<char>? strMemory, char
                 {
                     inQuotes = !inQuotes;
                 }
-
-                if (!inQuotes && separators.IndexOf(c) >= 0)
+                else if (!inQuotes && separators.IndexOf(c) >= 0)
```

Comment:
Wouldn't `Contains` be even faster than `IndexOf >= 0`? Perhaps also SearchValues would make sense here.

Reply:
Switched to Contains in commit 2. 

Thanks for the tip about SearchValues, I didn't know about that. I think I'm going to pass on that for now, but I'll keep that in the back pocket for the next time I see something like this.